lib/pbench/test/unit/server/test_user_auth.py

```diff
@@ -398,13 +398,13 @@ def test_invalid_logout(client, server_config):
             )
             assert logout_response.status_code == 200
 
-            # invalid token logout
+            # Logout using invalid token
+            # Expect 200 on response, since the invalid token can not be used anymore
```

Comment:
Of course if it was really an "invalid token" then it could never be used in the first place. More like, either the token was garbage or it's been deleted, and either way cannot be used to authorize access.